text-editors-ides.md

```diff
@@ -0,0 +1,7 @@
+# Text Editors and IDEs
+
+Here are some of the options LAA Developers can choose from when selecting an IDE/text editor to work with.
+
+| Name | Status | Usecase  | Notes  |
+|---|---|---|---|
+| Atom | Recommended | Default choice | Text editor with built in Git/GitHub integration. A range of plug in packages can be added such as an inbuilt terminal and markdown preview. https://atom.io/ https://atom.io/packages |
```

Comment:
What makes this the "Default choice"? Or are you just coming up against the template here? Would it be better to change the structure of the table for pages like this, where we aren't necessarily looking for convergence at all?

Reply:
Fair points - I've removed the 'Status' and 'Usecase' columns